lib/blockscore/candidate.rb

```diff
@@ -19,13 +19,13 @@ def hits
     end
 
     def search(options = {})
-      post "#{api_url}watchlists", options.merge(candidate_id: id)
+      post 'watchlists', options.merge(candidate_id: id)
     end
 
     private
 
-    def resource_member(member)
-      get "#{endpoint}/#{id}/#{member}", {}
+    def resource_member(target)
+      get member_endpoint + target, {}
```

Comment:
Yes - the tests all pass, and in my RSpec refactoring there will be specific tests in `Base`. The endpoint is a `Pathname` so adding segments returns another `Pathname` which joins with the slash.